app/src/main/res/layout/view_toolbar.xml

```diff
@@ -36,7 +36,7 @@
         android:gravity="bottom"
         android:orientation="horizontal"
         android:paddingStart="24dp"
-        android:paddingEnd="24dp"
+        android:paddingEnd="0dp"
```

Comment:
Couldn’t this be removed instead of setting it to `0`?

Reply:
Definitely could, I was just trying to appease the warning it gave me about including a paddingEnd for symmetry

Reply:
If that warning is from the Android Linter you can add `tools:ignore="warningid"` to ignore it